charts/kubewarden-controller/values.yaml

```
@@ -300,7 +300,7 @@ policy-reporter:
     image:
       registry: ghcr.io
       repository: kyverno/policy-reporter-ui
-      tag: 2.4.3
+      tag: 2.4.4
```

```suggestion
      tag: 2.4.3
```

https://github.com/kubewarden/helm-charts/issues/797